### PR TITLE
#87 feature: A buyer should be able to view their cart

### PR DESCRIPTION
**Pull request by sostene12**
#### What does this PR do?
- View the user's cart
#### Description of Task to be completed
- implement get the logged-in user cart
- implement cart overlay modal
#### How should this be manually tested?
- clone the repo using git clone repo's URL
- change branch `git ft-view-cart-87`
- create .env file  and paste variables from .env.example
- run `npm run dev`
- login as a buyer
- navigate to the `/cart` or click on the cart
#### What are the relevant pivotal tracker stories?
-  [A buyer should be able to view their cart](https://trello.com/c/7TsZ6ita)

### CODE REVIEW

**Comment by sostene12** [comment]
[A buyer should be able to view their cart](https://trello.com/c/7TsZ6ita/87-a-buyer-should-be-able-to-view-their-cart)

**Review by oliviertech212 — APPROVED** [review]

**Review by oliviertech212 — APPROVED** [review]

File: src/api/cartapi.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+				}
+				reject(error);
+			});
+	});
+};
+
+export const clearUseCart = (token) => {
+	return new Promise((resolve, reject) => {
+		axios
+			.delete(`${url}/api/cart/clear-cart/1`, {
```
Comment (wakaflorien) [inline src/api/cartapi.js]: -  the cart id here should not be static.

**Review by wakaflorien — CHANGES_REQUESTED** [review]

File: src/api/cartapi.js (hunk 1 above)
Comment (sostene12) [reply src/api/cartapi.js]: done

Comment (sostene12) [reply src/api/cartapi.js]: done

**Review by patsicko — APPROVED** [review]

**Review by Luciefifi — APPROVED** [review]

**Review by wakaflorien — APPROVED** [review]

**Pull request merged**